### ISSUE TITLE
run() leaves /tmp/pexpect.py

### ISSUE THREAD

**Issue by ScottSturdivant** [JasperVanDenBosch/fexpect#6]
First, let me just state that fexpect is fantastic.  I'm not quite sure why this isn't built into fabric, but it's great!

The problem I'm seeing is that if I do this as root:

```
with expecting(prompts):
    run(cmd)
```

It leaves /tmp/pexepect.py(c) and /tmp/fexpect\* files all owned as root.  Then if I run the same block as a normal user, I'm met with a permission denied error as it tries to put(pexpect) into /tmp.  I've hacked around this by using the following:

```
with expecting(prompts):
    run(cmd)
run('rm /tmp/{*expect.*')
```

Is it possible to have the context manager perform the cleanup?

**Comment by JasperVanDenBosch** [comment]
Hey, good point, thanks for submitting an issue on this. I think I fixed this by explicitly 'putting' the pexpect module with unrestrictive permissions. Could you try it out? The fix is now on pypi as the latest version, 0.2.post7

**Comment by ScottSturdivant** [comment]
Success!  Thank you so much for fixing that!
